app/bundles/CampaignBundle/Form/Type/CampaignEventAddRemoveLeadType.php

```diff
@@ -51,15 +51,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     /**
      * @return string
      */
-    public function getName()
+    public function getBlockPrefix()
     {
-        return 'campaignevent_addremovelead';
+        return self::class;
```

Comment:
If we do not need to keep the block prefix because it's used in CSS or JS then we can delete that method and let the parent class take over. If you check what it does int the AbstractType then it's very similar to this.

Reply:
How can I be sure its not used in CSS or JS? Is it sufficient to just search for the string name, like "campaignevent_addremovelead"?

Reply:
That's what I did. But I leave it there in the bundles I refactor and just rename the getName method to getBlockPrefix to be sure. See https://github.com/symfony/symfony/blob/3.0/UPGRADE-3.0.md#form where it is described with examples.